sympy/vector/coordsysrect.py

```diff
@@ -174,7 +220,7 @@ def __new__(cls, name, location=None, rotation_matrix=None,
 
         obj._parent_rotation_matrix = parent_orient
         obj._origin = origin
-
+        obj.transformation_equations = transformation_equations
```

Comment:
Store `transformation`, not `transformation_equations` (they are the same thing in the end).